numba/core/debuginfo.py

```diff
@@ -422,8 +422,12 @@ def _di_subroutine_type(self, line, function, argmap):
 
         for idx, (name, nbtype) in enumerate(argmap.items()):
             name = name.replace('.', '$')    # for gdb to work correctly
-            datamodel = self.cgctx.data_model_manager[nbtype]
-            lltype = self.cgctx.get_value_type(nbtype)
+            use_ty = nbtype
+            if isinstance(nbtype, types.Omitted):
+                resolver = self.cgctx.typing_context.resolve_value_type
+                use_ty = resolver(nbtype.value)
```

Comment:
hm... Maybe `types.Omitted` should use some dummy type for the value model

Reply:
Could just use an `ir.IntType(8).as_pointer()` or were you thinking more along the lines of creating a special dummy type?

Reply:
`i8*` works just fine. other choices are a empty struct (`llvmlite.ir.LiteralStructType([])`.